spring-cloud-gcp-storage/pom.xml

```diff
@@ -23,5 +23,14 @@
 			<groupId>com.google.cloud</groupId>
 			<artifactId>google-cloud-storage</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-core</artifactId>
```

Comment:
Should this be `optional` as well?

Reply:
We don't this dependency at all.
It is transitive with the `spring-integration-file`